packages/block-editor/src/store/selectors.js

```diff
@@ -617,7 +634,7 @@ export const getMultiSelectedBlocks = createSelector(
 		return multiSelectedBlockClientIds.map( ( clientId ) => getBlock( state, clientId ) );
 	},
 	( state ) => [
-		...getMultiSelectedBlockClientIds.getDependants( state ),
+		...getSelectedBlockClientIds.getDependants( state ),
```

Comment:
A bit unfortunate we have to pierce the abstraction, since `getMultiSelectedBlocks` only references `getMultiSelectedBlockClientIds`, not `getSelectedBlockClientIds`, but since `getMultiSelectedBlockClientIds` has no inspectable dependents, I see no other option. ¯\\\_(ツ)\_/¯